docs/modules/data-structures/pages/distributed-data-structures.adoc

```diff
@@ -8,119 +8,120 @@ All of these structures are available in Java, .NET, C++, Node.js, Python, and G
 
 [cols="20%a,40%a,20%a,20%a"]
 |===
-|Data structure |Description|Partitioned/Non-partitioned|AP/CP
+|Data structure |Description|Partitioned/Non-partitioned|Tolerance
```

Comment:
It's really about availability, consistency, and partition tolerance (CAP). CAP theorem states that a system can deliver two of the three aspects at one time.

Reply:
I've linked the column heading to the relevant section in the overview that gives a fuller explanation. WDYT?